src/driver/amdxdna/amdxdna_gem.c

```diff
@@ -835,10 +830,12 @@ amdxdna_drm_create_share_bo(struct drm_device *dev,
 
 	if (args->vaddr)
 		abo = amdxdna_gem_create_user_object(dev, args);
-	else if (xdna->use_cma)
+#ifdef AMDXDNA_DEVEL
```

Comment:
my only concern is that we now have to set iommu_off to use either cma or carvdout.

I think in the past even with iommu on, we still can leverage the carvdout.

In some bringup situation, we might want to test physical address regardless if iommu is on/off.